.eslintrc.json

```diff
@@ -74,32 +67,48 @@
         "id-blacklist": "off",
         "id-match": "off",
         "max-len": "off",
-        "linebreak-style": [
+        "linebreak-style": ["error", "unix"],
+        "no-underscore-dangle": "off",
+        "@nrwl/nx/enforce-module-boundaries": [
           "error",
-          "unix"
-        ],
-        "no-underscore-dangle": "off"
+          {
+            "allow": [],
+            "depConstraints": [
+              {
+                "sourceTag": "*",
+                "onlyDependOnLibsWithTags": ["*"],
+                "notDependOnLibsWithTags": ["type:app"]
+              },
+              {
+                "sourceTag": "type:feature",
+                "notDependOnLibsWithTags": ["type:integration"]
+              },
+              {
+                "sourceTag": "type:ui",
+                "notDependOnLibsWithTags": ["type:feature"]
+              },
+              {
+                "sourceTag": "scope:core",
+                "notDependOnLibsWithTags": ["type:ui", "type:feature"]
```

Comment:
We could add `integration`, or is it indirectly applied via `feature`?  (in which can we only keep `ui` )